nginx.agent.conf

```diff
@@ -42,10 +42,12 @@ http {
 
             proxy_pass http://log/;
         }
-
-        location /system/metrics {
-            include common/proxy-headers.conf;
-            proxy_pass http://metrics;
+        
+        location /system/v1/metrics/ {
```

Comment:
These are new services, so no customers are changing them. We want to version the /system/ URI to make it feel more like a single API.